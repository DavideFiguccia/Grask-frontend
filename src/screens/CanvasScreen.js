import React from 'react';
import { GraphEditor } from '../components/graph_editor/GraphEditor';

export class CanvasScreen extends React.Component {

    render() {
        return <div>
            <GraphEditor/>
        </div>
    }

}


