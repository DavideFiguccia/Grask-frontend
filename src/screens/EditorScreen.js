import React from 'react';
import { GraphEditor } from '../components/GraphEditor/GraphEditor';

export class EditorScreen extends React.Component {

    render() {
        return <div>
            <GraphEditor/>
        </div>
    }

}


