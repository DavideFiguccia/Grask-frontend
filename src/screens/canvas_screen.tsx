import React from 'react';
import '../components/graph_editor/graph_editor.css';
import '../components/graph_editor/graph_editor';

export class CanvasScreen extends React.Component {

    render(){
        return <div className='graph_editor.css'>
            <script src="../components/graph_editor/graph_editor.js"/>
        </div>
    }

}


