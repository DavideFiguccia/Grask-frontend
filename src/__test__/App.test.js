import React from 'react';

import { create } from "react-test-renderer";

import { CanvasScreen } from '../screens/CanvasScreen';

test("Matches the snapshot", () => {
    const button = create(<CanvasScreen />);
    expect(button.toJSON()).toMatchSnapshot();
  });
