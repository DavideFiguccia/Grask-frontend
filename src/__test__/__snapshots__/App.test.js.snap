// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<div>
  <div>
    <div
      className="diagram-component"
    />
  </div>
</div>
`;
